app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = []
+    for i in range(len(people)):
+        result.append(
+            Person(
+                people[i]["name"], people[i]["age"]
+            ).people[people[i]["name"]]
```

Comment:
You can add just a `Person` instance. You don't need to take `people` attribute from it
``` result.append(Person(....))```

Reply:
I did it like this (see below), but you said that I should write in one line that's why I change to one line **result.append(Person(people[i]["name"],people[i]["age"]).people[people[i]["name"]])**:
PRIOR CODE:
   result = []
   for i in range(len(people)):
       Person(people[i]["name"], people[i]["age"])
       result.append(Person.people[people[i]["name"]])

Reply:
I've done some changes. Could you look at it.